scripts/decredStakeVoting.md

```diff
@@ -3,45 +3,44 @@
 ### Keywords
 `unique`, `lucrative`, `investor`, `stake vote`, `no effort`, `risk-free`, `proof-of-stake`
 
-Decred’s unique stake voting system offers a functional and lucrative opportunity to investors. By participating in the proof-of-stake mining system, investors become stake voters helping to both secure the Decred blockchain and choose the direction of its evolution.
+Decred’s unique stake voting system offers a functional and lucrative opportunity to investors. By participating in the proof-of-stake mining system, investors become stake voters helping to secure the Decred blockchain, choose the direction of its evolution, and earn Decred at 15% APR with zero risk.
 
-Investors can become stake voters and begin earning Decred at 15% APR with zero risk by purchasing tickets.
+Stake voting requires no action by the stakeholder other than purchasing tickets.
 
 By purchasing a ticket, an investor locks a portion of their Decred funds equal to the current ticket price.
 
-The price of a ticket varies based on the number of tickets waiting in what is called the ticket pool. Ticket price is automatically adjusted to keep the size of the ticket pool stable. If the ticket pool contains more than the optimum 40,960 tickets, the price to purchase a ticket will increase. An increased ticket price will discourage ticket buying. A discouraged ticket market will allow the number of tickets in the pool to shrink. If there are too few tickets in the ticket pool, the price will decrease to kick-start investment and grow the pool.
+The ticket price is automatically adjusted so that the right amount of tickets remain in the ticket pool. If the ticket pool contains more than the optimum 40,960 tickets, the price to purchase a ticket will increase. An increased ticket price will discourage ticket buying an allow the number of tickets in the pool to shrink. If there are too few tickets in the ticket pool, the price will decrease to kick-start investment and fill the pool.
 
-A ticket in the pool has a 99.5% chance of being drawn within 5 months. On average, a ticket will only wait in the pool for 30 days. For the fraction of a percent of tickets that expire, the corresponding staked Decred are unlocked, ready to be spent on a replacement ticket.
+A ticket in the pool has a 99.5% chance of being drawn within 5 months. On average, a ticket will only wait in the pool for a month. For the fraction of a percent of tickets that expire, the corresponding staked Decred are unlocked and returned to the user who can then decide whether to spend them on another ticket or something else.
 
-Tickets that are drawn from the pool are chosen randomly. Using random entropy from the blockchain, 5 tickets are drawn.
+Tickets are drawn from the pool randomly, using random entropy generated by the blockchain. With each new block, 5 tickets are drawn to vote.
```

Comment:
Switch this paragraph with the previous paragraph so that we talk about votes before talking about how long the user has to wait to vote.